util/src/bin/dns.rs

```diff
@@ -505,6 +516,99 @@ async fn handle_request(
             println!("; sending delete-record: {name} {class} {ty} from {zone}");
             client.delete_by_rdata(rdata, zone).await?
         }
+        Command::FetchKeys(opt) => {
+            let zone = zone.unwrap_or_else(Name::root);
+            let record_type = RecordType::DNSKEY;
+
+            println!("; querying {zone} for key-signing-dnskeys, KSKs");
+
+            let response = client.query(zone, class, record_type).await?;
+            let response = response.into_message();
+
+            cfg_if! {
```

Comment:
Not sure what `cfg_if!{}` adds here that couldn't be done with simple blocks:

```
#[cfg(not(feature = "dnssec-ring"))]
{
   ..
}
```

Reply:
true, not sure why I didn't do that. I can blackout the cfg_if dependency as well.